### ISSUE TITLE
"Episodes watched" count not progessing.

### ISSUE THREAD

**Issue by Smgit5** [saikou-app/saikou#420]
### Summary

Suppose I started watching an anime and completed first three episodes. The individual progress for that specific anime is progressing. But the stat for "Episodes watched" is still at 0. Also the stat is not updating in my anilist account, so I'm not sure the problem is in the app or in my anilist account. After uninstalling & reinstalling the app the "Episodes watched" count is shown correct in both the app & anilist account (i.e before uninstalling was 0 -> after uninstalling is 3).

### Reproduction Steps

1. Watch more than one episodes for any anime.
2.  Check the "Episodes watched" count at the top of the app interface, whether it is progressing or not.
3. Also go to anilist setting -> anime list -> overview. Check the "Episodes watched" count in your anilist account.
4. I attached the screenshots of the app and my anilist account.
![Screenshot_20221013-005034](https://user-images.githubusercontent.com/94289238/195429999-21c87ac5-4650-4557-94b5-53a1b6cef247.png)
![Screenshot_20221013-005139](https://user-images.githubusercontent.com/94289238/195430305-69d55eed-c946-4378-8db1-7f0b74dbc10d.png)

### Expected Result

I watched a total of 7 episodes combining three different animes (screenshots provided). So the "Episodes watched" count should be 7.

![Screenshot_20221013-005034](https://user-images.githubusercontent.com/94289238/195433983-2a12125a-2530-4f29-9e07-91a46456a5c6.png)
![Screenshot_20221013-005139](https://user-images.githubusercontent.com/94289238/195433996-f82baf79-f17e-4e92-aec3-0b38f8ede1be.png)

### Actual Result

The current "Episodes watched" count is 5 instead of 7. (Screenshot provided)

![Screenshot_20221013-005139~2](https://user-images.githubusercontent.com/94289238/195434566-1ac908af-f773-4353-9ced-01454f909d29.png)
![Screenshot_20221013-005034~2](https://user-images.githubusercontent.com/94289238/195434577-67d4a65e-79c9-4033-8947-1577af15f138.png)

### Device Information

Saikou Version: 1.2.0.22
Device: motorola guamp
Architecture: aarch64
OS Version: REL 11 (30)

### Checklist

- [X] I have searched the open AND closed issues for any duplicates.

### Additional Information

After uninstalling & reinstalling the "Episodes watched" count is shown correct.
I used Saikou v1.2.0.22 & as well as v1.2.0.21, both beta. The bug is present in both versions. I don't know about other versions.

**Comment by brahmkshatriya** [comment]
That auto updates after every 48hrs, you force update it through your anilist settings

**Comment by Smgit5** [comment]
I have already tried force update through anilist setting and restarted the app. The stat didn't change.

**Comment by Adolar0042** [comment]
Have you disabled updating the anime automatically? Long-Press on the "Watching" button when on the titles site and go into any episode and enable it

**Comment by Smgit5** [comment]
Oh, actually when I first installed the app and started watching kochikame, on the first episode I selected "NO" in the prompt asking to set auto progress as "yes" or "no". But on the 2nd episode I selected "yes". Also after uninstalling & reinstalling & logging in I went into settings and turned off "Ask for each anime individually". (Screenshots). But still the same problem.

![Screenshot_20221013-014236~2](https://user-images.githubusercontent.com/94289238/195440789-d1e9df90-b666-4b52-851b-76d928b1dd9f.png)
